hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/PipelineEndpoint.java

```diff
@@ -77,7 +77,7 @@ public Response getPipelines() {
       UUID pipelineId = pipeline.getId().getId();
       List<String> datanodes = new ArrayList<>();
       PipelineMetadata.Builder builder = PipelineMetadata.newBuilder();
-      pipeline.getNodes().forEach(node -> datanodes.add(node.getHostName()));
+      pipeline.getNodes().forEach(node -> datanodes.add(node.getHostName() + "/" + node.getUuid()));
```

Comment:
I agree with @myskov, it would be better to have list of `DatanodeDetails` as `datanodes`, instead of string and attach the UUID with a slash.